csd/descriptor/service.yaml

```diff
@@ -119,8 +120,8 @@ roles:
   logging:
     dir: /var/log/hwx-registry
     filename: server.log
-    loggingType: other
     modifiable: false
+    loggingType: log4j
```

Comment:
dropwizard actually uses logback, but it is easier to convert the log4j.properties into yaml using envsubst.